pkg/providers/version/version.go

```diff
@@ -32,19 +28,16 @@ import (
 
 const (
 	kubernetesVersionCacheKey = "kubernetesVersion"
-	// Karpenter's supported version of Kubernetes
-	// If a user runs a karpenter image on a k8s version outside the min and max,
-	// One error message will be fired to notify
-	MinK8sVersion = "1.23"
-	MaxK8sVersion = "1.30"
+	MinK8sVersion             = "1.26"
```

Comment:
Give a reason for just supporting 1.26~1.31.1 k8s

Reply:
For the max version, I just choose the max k8s release version.
For the min version, I just choose one not so old

Reply:
Is MinK8sVersion documented, or is it subjective?

Reply:
I updated, please check again